app/domain/authentication/authn_jwt/signing_key/create_signing_key_interface.rb

```diff
@@ -0,0 +1,75 @@
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # Factory that returns the interface implementation of FetchSigningKey
+      CreateSigningKeyInterface ||= CommandClass.new(
```

Comment:
why is this called `CreateSigningKeyInterface` and not `CreateSigningKeyFactory` or `CreateSigningKeyInstance`?

Reply:
Done